metricbeat/module/kubernetes/state_container/state_container.go

```diff
@@ -0,0 +1,72 @@
+package state_container
```

Comment:
I'm starting to think this should be `container_state` as the namespace is probably `container.state`?